src/controls/GridControl/_grid.scss

```diff
@@ -1,3 +1,13 @@
+%modal-button {
+  border-radius: 32px;
+  padding: 16px;
```

Comment:
can these padding props be replaced with the 'padding' mixin? After marking its arguments as optional